### PR TITLE
hass(improvement): stop specific values from populating

### PR DESCRIPTION
**Pull request by billiaz**
Some values cause too much chatter in Hass, we remove these from discovery populating

### CODE REVIEW

**Review by robertsLando — CHANGES_REQUESTED** [review]
I prefer to move the check in `discoverValue` function

**Comment by billiaz** [comment]
wouldn't that iterate all values and just return, instead of stoping earlier?

**Comment by robertsLando** [comment]
@billiaz discoverValue could also run 'on fly' and in that case this values will be discovered because you rely on the filter done by the prioritized cc function.

**Comment by billiaz** [comment]
@zwave-js-bot fix lint

File: lib/Gateway.js [hunk 1]
```diff
@@ -1608,6 +1609,14 @@ Gateway.prototype.discoverClimates = function (node) {
...
   const valueId = node.values[vId]
 
+  // List of values
+  const ignoreDiscovery = {
+    Meter: ['previousValue', 'deltaTime']
+  }
+  if (ignoreDiscovery[valueId.commandClassName]) {
+    if (ignoreDiscovery[valueId.commandClassName].includes(valueId.property))
+      return
+  }
```
Comment (robertsLando) [inline lib/Gateway.js:1619]: I like this but for now there are much other cc we are ignoring, IMO this check should be done inside the CC switch-case, like `if(property === x || property === y) return`

**Review by robertsLando — CHANGES_REQUESTED** [review]

File: lib/Gateway.js (hunk 1 above)
Comment (aretakisv) [reply lib/Gateway.js:1619]: thought of. looking if we have second class, otherwise i will move it under. the idea was to cover all

**Comment by robertsLando** [comment]
With zwavejs 7 deltaTime and previous value will not be available anymore so this can be closed

**Pull request closed without merging**